core/app/models/spree/taxon/paperclip_attachment.rb

```diff
@@ -18,4 +18,8 @@ module Spree::Taxon::PaperclipAttachment
   def icon_present?
     icon.present?
   end
+
+  def attachment_partial_name
+    'paperclip_attachment'
```

Comment:
Maybe it's better to have only `paperclip` here since it could be used also in other context. 

We can use 

```ruby
<%= render "spree/admin/taxons/attachment_forms/#{f.object.attachment_partial_name}", f: f %>
```

So we'll be able to have a folder for the forms and one for the show views (if needed in the future). This is exactly the same pattern we are using for payments, see [here](https://github.com/solidusio/solidus/tree/df79573cc09335dd8d2b193d8f54cd0e03a2cc05/backend/app/views/spree/admin/payments). WDYT?